### ISSUE TITLE
Can this driver support Windows server 2003?

### ISSUE THREAD

**Issue by BruceMok** [camthesaxman/win9x_vm_display_driver#3]
Base on QEMU-KVM,  guest OS is Windows server 2003,  graphic display device is QXL, can use this driver?

**Comment by camthesaxman** [comment]
No, Windows Server 2003 is a Windows NT operating system. This driver is only for Windows 95, 98, and Me. VirtualBox and VMware have official support for Windows Server 2003 guests, so something like this is not needed.
